src/tabs/tabs.jsx

```diff
@@ -101,7 +103,7 @@ let Tabs = React.createClass({
         }
 
         tabContent.push(tab.props.children ?
-          React.createElement(TabTemplate, {
+          React.createElement(tabTemplate || TabTemplate, {
```

Comment:
I have one final request before merging. Can you move `TabTemplate` to `getDefaultProps`?